tests/s25Main/integration/testEconomyMode.cpp

```diff
@@ -129,8 +129,8 @@ BOOST_FIXTURE_TEST_CASE(EconomyMode3Players, WorldWithGCExecution3P)
     newWorld.econHandler->UpdateAmounts();
     for(const auto& teamPair : boost::combine(econTeams, econTeamsAfter))
     {
-        const auto& before = teamPair.get_head();
-        const auto& after = teamPair.get_tail().get_head();
+        const EconomyModeHandler::EconTeam& before = boost::get<0>(teamPair);
+        const EconomyModeHandler::EconTeam& after = boost::get<1>(teamPair);
```

Comment:
Just a hint (no need to change): You can rely on ADL here and use `get<0>(teamPair)`. This is less explicit and has the benefit that one could switch to a combine/zip iterator that returns std:: types

Reply:
Ok, thanks